scipy/linalg/basic.py

```diff
@@ -1257,10 +1257,14 @@ def pinv(a, cond=None, rcond=None, return_rank=False, check_finite=True):
         Matrix to be pseudo-inverted.
     cond, rcond : float, optional
         Cutoff for 'small' singular values in the least-squares solver.
-        Singular values smaller than ``rcond * largest_singular_value``
-        are considered zero.
-        If None, it is set to ``np.finfo(a.dtype).eps``.
-        If `a` is an array of integers, it is set to ``np.finfo('float64').eps``.
+        Singular values smaller than ``max(M, N) * eps`` are considered zero
```

Comment:
The ``max(M, N) * eps`` looks more like a description of the default, than a description of the behavior of the parameter. So instead of saying "Singular values smaller than ..." it should probably say "If cond and rcond are None, a value of max(M,N)*eps is used".

`cond` gets passed directly to [lstsq](https://docs.scipy.org/doc/scipy/reference/generated/scipy.linalg.lstsq.html), where it describes this as:

> Cutoff for ‘small’ singular values; used to determine effective rank of a. Singular values smaller than rcond * largest_singular_value are considered zero.

This seems like a clearer wording of the behavior than what is in this PR. I'd prefer that this docstring say the same thing as `lstsq` (same goes for `pinv2` and `pinvh`). After describing how `rcond` is used, then it should mention the default max(M,N)*eps.

Reply:
Indeed, I apparently cut too much including the `if None` part.